spec/requests/groups_controller_spec.rb

```diff
@@ -547,6 +547,23 @@ def expect_type_to_return_right_groups(type, expected_group_ids)
 
       expect(members.last['added_at']).to eq(first_user.created_at.as_json)
     end
+
+    it "can sort items" do
+      sign_in(user)
+      group.update!(visibility_level: Group.visibility_levels[:logged_on_users])
+      other_user = Fabricate(:user)
+      group.add_owner(other_user)
+
+      get "/groups/#{group.name}/members.json"
+
+      expect(response.parsed_body["members"].map { |u| u["id"] }).to contain_exactly(other_user.id, user.id)
+      expect(response.parsed_body["owners"].map { |u| u["id"] }).to contain_exactly(other_user.id)
```

Comment:
```suggestion
      expect(response.parsed_body["members"].map { |u| u["id"] }).to eq([other_user.id, user.id])
      expect(response.parsed_body["owners"].map { |u| u["id"] }).to eq([other_user.id])
```

Note that the `contain_exactly` disregards ordering so we can't use that here if we're need to ensure that the results are ordered correctly.